app/src/main/java/net/squanchy/navigation/HomeActivity.java

```diff
@@ -235,10 +265,10 @@ protected void onSaveInstanceState(Bundle outState) {
     protected void onStop() {
         super.onStop();
 
-        if (proximityServiceRadarStarted) {
+        /*if (proximityServiceRadarStarted) {
```

Comment:
How come? Is this supposed to be taken out? If so please delete it, otherwise uncomment